### PR TITLE
ENH: Add massich to python

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

File: mentors/python/m.json [hunk 1]
```diff
@@ -7,4 +7,12 @@
         "avatar_url": null,
         "bio": "I use Python every day in my role as a Data Engineer. My last name is pronounced 'Koontz'."
     }
```
Comment (SleeplessByte) [inline mentors/python/m.json]:
```suggestion
    },
```

**Comment by massich** [comment]
@SleeplessByte done

**Comment by massich** [comment]
@SleeplessByte done, feel free to push in the PR.

**Comment by SleeplessByte** [comment]
Welcome to Python 🤝

**Pull request merged**